### PR TITLE
add test to TitleWithDescription #126

### PR DESCRIPTION
**Pull request by Mygal-Alex**
image
image

### CODE REVIEW

**Review by AndriiNasadiuk — APPROVED** [review]

**Review by marianamartsy — APPROVED** [review]
👍

**Review by LeonidShamarin — APPROVED** [review]

**Review by ArtemHolikov — APPROVED** [review]
Good

**Comment by vlad-khrychov** [comment]
Add `close #126` to description

File: tests/unit/components/title-with-description/TitleWithDescription.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  it('should render without title and description', () => {
+    render(<TitleWithDescription />)
+
+    const title = screen.queryByText(/TEST_TITLE/)
+    const description = screen.queryByText(/TEST_DESCRIPTION/)
+
+    expect(title).not.toBeInTheDocument()
+    expect(description).not.toBeInTheDocument()
+  })
```
Comment (vlad-khrychov) [inline tests/unit/components/title-with-description/TitleWithDescription.spec.jsx]: I think this test we can delete. It is like `expect(2).not.toEqual(3)`.
```suggestion
```

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

**Pull request merged**